src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Stylesheets with a .scss extension won't work directly in browsers. You need to link to the compiled .css file instead. Ensure that you compile `index.scss` to `index.css` and link to the latter.